doc/api/util.md

```diff
@@ -398,6 +398,83 @@ util.inspect.defaultOptions.maxArrayLength = null;
 console.log(arr); // logs the full array
 ```
 
+## util.promisify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function}
+
+Takes a function following the common Node.js callback style, i.e. taking a
+`(err, value) => ...` callback as the last argument, and returns a version
+that returns promises.
```

Comment:
sync functions that throw reject the promise (`new Promise` catches it) and sync functions that return a value resolve it with that.

In other words, it works just fine.